models/review.go

```diff
@@ -283,10 +298,18 @@ func SubmitReview(doer *User, issue *Issue, reviewType ReviewType, content strin
 			return nil, nil, ContentEmptyErr{}
 		}
 
+		// Official status of review updated at every submit in case settings changed
+		official, err := isOfficialReviewer(sess, issue, doer)
```

Comment:
`Review.Official` is not being set when the review is created above in line 284 (e.g. direct approval without commenting code).

Reply:
It was done in `createReview`, but the updated code shall be more clear. An approved or rejected review always go to function `SubmitReview`, and the update of official attribute is handled there now.